src/main/webapp/search-results.html

```diff
@@ -18,16 +15,18 @@
     <meta charset="UTF-8">
     <title>Results</title>
     <link rel="stylesheet" type="text/css" href="search-results.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
   </head>
   <body onload="getSearchResults()">
-    <div id="content">
-        <form id="search-form" onsubmit="return redirectToResults();">
-            <input type="text" id="topic-search-box" placeholder="What do you need help with today?">
-            <button type="submit">Go</button>
+    <div id="content" class="container-fluid" style="text-align: center;">
+        <form id="search-form-results" onsubmit="return redirectToResults();">
```

Comment:
I think that you could just say onsubmit="redirectToResults();" here. Same thing would go for search.html.

Reply:
I think I need that the `return` because the form will not stop submission until it finds a return statement (in which case it won't redirect the page).